### ISSUE TITLE
[Bug Report] VExpansionPanelContent open transition 

### ISSUE THREAD

**Issue by JFFilip** [vuetifyjs/vuetify#4561]
### Versions and Environment
Vuetify: 1.0.17 (also did not work in 1.1.3)
Vue: nuxt-edge: latest
Browsers:  Chrome: Version 67.0.3396.99
OS: windows 10

### Steps to reproduce

Create VExpansionPanel with multiple children (~80+) a then try to reopen them. Try to leave browser tab, revisit and continue with reopen panels. It happened to me like 1 of 30 reopen.

### Expected Behavior
VExpansionPanelContent should open properly

### Actual Behavior
Sometimes VExpansionPanelContent body is stuck in transition and is still hidden (https://prnt.sc/k4n5p8)

It looks like [fn](https://github.com/vuetifyjs/vuetify/blob/dev/src/components/transitions/expand-transition.js#L5) is called but [fn](https://github.com/vuetifyjs/vuetify/blob/dev/src/components/transitions/expand-transition.js#L19) is not.

### Reproduction Link
[codepen](https://codepen.io/anon/pen/ejYdGo?editors=0011) I tried to reproduce it in codepen, but there is no problem

**Comment by JFFilip** [comment]
I found a solution. I had watcher on variable that control open state of VExpansionPanelContent and when this variable has been changed, I send ajax to load data into body of VCard in panel. Before the content load, VCard in panel body was empty and transiton on panel keep its transition state unfinished. Panel after update from ajax still has its unifinsed transition state. Dunno if this is issue or not. [Codepen](https://codepen.io/anon/pen/PBoWgG?editors=1011) with empty VCard in panel body.
